language/diem-tools/writeset-transaction-generator/src/release_flow/unit_tests/release_tests.rs

```diff
@@ -26,20 +29,21 @@ fn release_test() {
     for i in 0..10 {
         let mut module = empty_module();
         module.identifiers[0] = Identifier::new(format!("test_{:?}", i)).unwrap();
-        let compiled_module = module.freeze().unwrap();
+        BoundsChecker::verify_module(&module).expect("invalid bounds in module");
```

Comment:
Same as the other writeset check, it should just verify completely

Reply:
Just wanted to point one thing out: prior to this pull request and your comment here, `empty_module` would return a module that would fail full verification, due to an `EMPTY_CODE_UNIT` error. In response to your suggestion here, I updated `empty_module` to generate a single `Ret` instruction in its code unit, which allows it to verify.